### ISSUE TITLE
Something in Firebase prevents from updating to Firebase/AdMob ~> 8.3.0

### ISSUE THREAD

**Issue by Idomo** [firebase/firebase-ios-sdk#7833]
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 12.4 (12D4e)
  * Firebase SDK version: 7.9.0
  * Installation method: `CocoaPods`
  * Firebase Components: (Auth, Core, Database, Storage, Crashlytics, Analytics, AdMob)

#### Steps to reproduce:

I've got messages that I should upgrade Google AdMob SDK to V8.
I've started with upgrading All Firebase dependencies to the most up-to-date version (7.9.0), worked fine.
Now when running `pod outdated` I'm getting this:
```
Google-Mobile-Ads-SDK 7.69.0 -> 7.69.0 (latest version 8.3.0)
```
When trying to force this to be 8.3.0 I'm getting this:
```
[!] CocoaPods could not find compatible versions for pod "Firebase/Analytics":
  In Podfile:
    Firebase/Analytics (~> 7.9.0)

None of your spec sources contain a spec satisfying the dependency: `Firebase/Analytics (~> 7.9.0)`.

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.
```

Tried comment out `Firebase/Analytics` and got the same error on `Firebase/Crashlytics` and so on about `Firebase/Auth` etc.

**Bottom line:**
It seems like `Firebase/AdMob` newest version doesn't compatible with other `Firebase` repos.

#### Relevant Code:

Podfile:
```
	pod 'Firebase/Core', '~> 7.9.0'
	pod 'Firebase/Storage', '~> 7.9.0'
	pod 'Firebase/Database', '~> 7.9.0'
	pod 'Firebase/Auth', '~> 7.9.0'
	pod 'Firebase/Crashlytics', '~> 7.9.0'
	pod 'Firebase/Analytics', '~> 7.9.0'
	pod 'Firebase/AdMob' #, '~> 7.69.0'
	pod 'SDWebImage', '~> 5.6.1'
	pod 'FirebaseUI/Storage', '~> 9.0.0'
	pod 'GoogleSignIn' #, '~> 5.0.2'
```

**Comment by paulb777** [comment]
tl;dr: Replace `pod 'Firebase/AdMob'` with `pod 'Google-Mobile-Ads-SDK'`.

Background, see docs at https://firebase.google.com/docs/ios/setup?authuser=0#initialize-firebase. Usage of `Firebase/AdMob` is no longer recommended since `Google-Mobile-Ads-SDK` releases at a different cadence than the other Firebase subspecs. It is especially exacerbated right now since `Google-Mobile-Ads-SDK` recently did a breaking change major release. 

The next Firebase major release will either update the `Admob` subspec to 8.x or completely remove it.

**Comment by Idomo** [comment]
Got it, thanks.

**Comment by paulb777** [comment]
Great! I'll leave this issue open to track making the Firebase podspec update for the 8.0.0 release.

**Comment by paulb777** [comment]
Current plan is to remove the Firebase/AdMob subspec in favor of directly using the `Google-Mobile-Ads-SDK` podspec.
